react-app-rewired/headers/csps/addressValidation.ts

```diff
@@ -0,0 +1,5 @@
+import type { Csp } from '../types'
+
+export const csp: Csp = {
+  'connect-src': ['https://api.proxy.shapeshift.com/api/v1/validate/'],
```

Comment:
we can probably rename this to just `shapeshiftProxy` and leave path as `/api/v1` in case we add anything else to our proxy

Reply:
Good call, actioned in https://github.com/shapeshift/web/pull/6626/commits/cb690dc240ef98e932939b571ba895f2d0839f40.